torch/quantization/quantize_fx.py

```diff
@@ -141,10 +141,12 @@ def create_node(self, kind : str, target : Target,
 
 def _prepare_fx(model: torch.nn.Module, qconfig_dict: Any,
                 prepare_custom_config_dict: Dict[str, Any] = None,
+                equalization_qconfig_dict: Dict[str, Any] = None,
```

Comment:
Should we add this to the end with an `_` prefix since this is still in development? @vkuzo what are your thoughts?

Reply:
yeah originally we thought of keeping it in the `qconfig_dict` but decided to separate it out to make it more explicit since equalization is a different step compared to quantization.

Reply:
We can move the equalization + quantization observers into EqualizationQConfig. So we'd have the following fields - 
`activation`, `weight`, `activation_eq` and `weight_eq`. @vkuzo would this make the code in `prepare` mode complex since we have to check for these fields for all nodes now?

Reply:
> yeah originally we thought of keeping it in the qconfig_dict but decided to separate it out to make it more explicit since equalization is a different step compared to quantization.

makes sense, thanks for explaining, never mind